config/core/test-configs.yaml

```diff
@@ -3110,3 +3120,7 @@ test_configs:
     test_plans:
       - baseline
       - smc
+
+  - device_type: fvp
+    test_plans:
+      - kselftest-seccomp
```

Comment:
Probably good to add a baseline test here as well, it helps if we can check if the platform booted before checking individual testsuites.